deployment/artifacts/lynx.json

```diff
@@ -1462,12 +1639,12 @@
                     "outputs": []
                 }
             ],
-            "tx_hash": "0x6495205ffa4f622cef2824c31b615812934379db8a6766e345881f5ac635f237",
-            "block_number": "40514266",
+            "tx_hash": "0xc2fd6f00ad2881cf3fd6b3440746dda9811c2cde625ef6c6d9acf47886bf5b77",
+            "block_number": 42479020,
```

Comment:
Block numbers seem to be now int literals, rather than strings, which is fine in itself, but I'm wondering about potential issues when parsing the registry.

Reply:
That's strange that it was a string before. I think that might be a bug with retroactively obtaining the deployment information - that was functionality that was added after a deployment was done (and a registry without that information already generated) via the `get_creation_info` - https://github.com/nucypher/nucypher-contracts/blob/main/deployment/legacy.py#L43. I think that line needs to cast the value to an int. 
7c515233f4d. Filed #194 - it may/may not apply to `mainnet` deployment if we merge the new deployment registry with the `mainnet` legacy registry (not sure if applicable or not).

In any case, I think we are good:
- On the `nucypher-contracts`-side, the `RegistryEntry` object expects `block_number` to be an `int` - https://github.com/nucypher/nucypher-contracts/blob/main/deployment/registry.py#L25, so writing it out it should be an int.
- On the `nucypher`-side, it doesn't care about block_number - https://github.com/nucypher/nucypher/blob/v7.0.0/nucypher/blockchain/eth/registry.py#L331